packages/samples/headless-react/src/components/dictionary-field-context/dictionary-field-context.fn.ts

```diff
@@ -0,0 +1,12 @@
+import {buildDictionaryFieldContext} from '@coveo/headless';
+import {useContext} from 'react';
+import {AppContext} from '../../context/engine';
+
+export function DictionaryFieldContext() {
+  const {engine} = useContext(AppContext);
+  const ctx = buildDictionaryFieldContext(engine!);
+
+  ctx.set({price: 'usd', city: 'boston'});
```

Comment:
I don't think that's a particularly good "real life" example/sample. 

I would poke someone from commerce team for this.

Reply:
@francoislg @dlafreniere what would you suggest?

Reply:
```
{ "price_currency": "fr", "price_discounted": "fr" }
```
is from a real client, but the example isn't so bad

Reply:
Thanks Devin,